webapp/src/index.tsx

```diff
@@ -7,16 +7,25 @@ import manifest from './manifest';
 // eslint-disable-next-line import/no-unresolved
 import {PluginRegistry} from './types/mattermost-webapp';
 
+const globalStats = {
+    Start: Date.now(),
+    NumDispatches: 0,
+};
+
 export default class Plugin {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
     public async initialize(registry: PluginRegistry, store: Store<GlobalState, Action<Record<string, unknown>>>) {
         window.webappDebugStore = store;
+        store.subscribe(() => {
```

Comment:
Should we unsubscribe in case we update the plugin afterwards? I don't know how much we care about leaking these subscriptions

Reply:
Yep, I can add that -- though `deinitialize` isn't "reliable". It works if you explicitly disable a plugin, but not if you just upload a replacement. I don't think the leak here will matter (compared to the issue being debugged at present!), but we could always give explicit instructions.